src/components/Game.vue

```diff
@@ -8,6 +8,7 @@
     <div id="errors"></div>
     <div class="row mt-2">
       <div class="col-md-6 order-2 order-md-1">
+        <PlayerInfo :player='data.players[this.player]' />
```

Comment:
This can be removed, and `orderedPlayers` can be changed.

Instead of ` const indexes = data.turnOrder.concat(data.passedPlayers);`, you can do:

`let turnOrder = data.turnOrder;

if (this.player !== undedined) {
   turnOrder = turnOrder.slice(this.player).concat(turnOrder.slice(0, this.player - 1));
}

const indexes = turnOrder.concat(...)
```